frame/transaction-payment/rpc/src/lib.rs

```diff
@@ -48,6 +48,20 @@ pub trait TransactionPaymentApi<BlockHash, ResponseType> {
 		encoded_xt: Bytes,
 		at: Option<BlockHash>,
 	) -> RpcResult<FeeDetails<NumberOrHex>>;
+
+	#[method(name = "payment_weightToFee")]
+	fn query_weight_to_fee(
+		&self,
+		weight: sp_weights::Weight,
+		at: Option<BlockHash>,
+	) -> RpcResult<NumberOrHex>;
+
+	#[method(name = "payment_lengthToFee")]
+	fn query_length_to_fee(
+		&self,
+		length: u32,
+		at: Option<BlockHash>,
+	) -> RpcResult<NumberOrHex>;
```

Comment:
New RPCs are discouraged. Having the runtime api is enough. People can call the state_call rpc to call the api.

Reply:
Good point, RPC seems unnecessary. Removed in ee6f399.